Improve graphql client bbes (#3825)
ballerina-platform/ballerina-distribution: examples/graphql-client-query-endpoint/graphql_client_query_endpoint.md

# GraphQL client - Query GraphQL endpoint

> The `graphql:Client` can be used to connect and interact with a GraphQL server.  A `graphql:Client` is created by passing the URL of a GraphQL service endpoint. The `execute` method is used to execute a GraphQL operation. This method requires the GraphQL `document` as a required argument and takes a map of `variables` and an `operationName` as arguments optionally, in case the document contains any variables or contains more than one operation. Use the GraphQL client to execute the `Query` and `Mutation` operations on a GraphQL service.

Reviewer: ```suggestion
The `graphql:Client` allows connecting and interacting with a GraphQL server.  A `graphql:Client` is created by passing the URL of a GraphQL service endpoint. The `execute` method is used to execute a GraphQL operation. This method requires the GraphQL `document` as a required argument and takes a map of `variables` and an `operationName` as arguments optionally, in case the document contains any variables or contains more than one operation. Use the GraphQL client to execute the `Query` and `Mutation` operations on a GraphQL service.
```
